src/App.css

```diff
@@ -140,3 +140,72 @@
   line-height: 70px;
   margin-bottom: 10px;
 }
+
+.input {
+  width: 250px;
```

Comment:
In real life it should be max-width or width 100% with depending on container or different width for different resolutions